### DISCUSSION TITLE
HB DAC+ Pro

### DISCUSSION THREAD

**Discussion by HunLucas** [rern/rAudio#490] [General]
Hello everyone!

I have a dac+ dps unfortunately this isn't support hardware volume control. If i buy a dac+ pro, which have hw volume control, can I use the volume controller on the WEB UI? or I can olny set with the command line commands?

Thank you!

**Comment by EarsofTin** [comment]
I have a Hifiberry DAC+and AMP2 that I can control volume from the UI. I think the only difference from the DAC+Pro is the separate clock on the Pro.

**Reply by HunLucas** [reply]
Thanks for your answer.
